backend/lucene/pom.xml

```diff
@@ -44,6 +44,10 @@
             <groupId>org.apache.lucene</groupId>
             <artifactId>lucene-facet</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.carrotsearch</groupId>
+            <artifactId>hppc</artifactId>
+        </dependency>
```

Comment:
Please don't use dependency management, see my main comment above.

Reply:
hmmmmmmmm. So with Lucene 9.5 this becomes a `<scope>runtime</scope>` dependency in the `lucene-facet` and enforcer plugin doesn't catch the version mismatch...

Reply:
What if we use the runtime scope ourselves? It would be fine as we definitely won't expose this dependency through our API.

Reply:
Mmm, I guess we cannot since we use types from it in our own facets - the compilation of those would fail, right?

Reply:
Right. Well, I don't know. Good luck :D